pvlib/pvsystem.py

```diff
@@ -1177,6 +1176,15 @@ def _parse_raw_sam_df(csvdata):
 
     df.index = parsedindex
     df = df.transpose()
+    if 'ADRCoefficients' in df.index:
+        ad_ce = 'ADRCoefficients'
+        df.loc[ad_ce] = df.loc[ad_ce].map(lambda x:
+                                          np.array([float(s) for s in
+                                                    [','.join(x[1:-1]
+                                                              .split())]
+                                                    [0].split(',')]))
```

Comment:
I think this works: lambda x: list(map(float, x.strip(' []').split(' ')))

not sure if there are pros/cons to adding np.array around the list. 

also, you don't have to stay justified with the opening parenthesis. you can make only a 4 space indent on the line below the opening parenthesis. that alone would help readability!

I'd get rid of the empty else block, too.

other than that, I think you only need to add documentation and then we'll be set.